<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DANodeLinkItemSettingWidget</class>
 <widget class="QWidget" name="DANodeLinkItemSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>212</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Node Link Item Setting</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="3" column="0">
    <widget class="QLabel" name="labelFrontEndPointStyle">
     <property name="text">
      <string>front style</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="DA::DAPenEditWidget" name="penEditWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>15</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBoxLinkStyle"/>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="comboBoxEndStyle"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelPen">
     <property name="text">
      <string>pen:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="comboBoxFrontStyle"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelLinkStyle">
     <property name="text">
      <string>link style:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelEndEndPointStyle">
     <property name="text">
      <string>end style</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelEndpointSize">
     <property name="text">
      <string>end point size</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="spinBoxEndpointSize"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DA::DAPenEditWidget</class>
   <extends>QWidget</extends>
   <header>DAPenEditWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
